packages/bpk-component-nudger/src/themeAttributes-test.js

```diff
@@ -0,0 +1,35 @@
+/*
+  * Backpack - Skyscanner's Design System
+  *
+  * Copyright 2017 Skyscanner Ltd
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License");
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  *   http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+
+import themeAttributes from './themeAttributes';
+
+describe('themeAttributes', () => {
+  it('should export the correct theme attributes', () => {
+    expect(themeAttributes).toEqual([
+      'buttonSecondaryTextColor',
+      'buttonSecondaryHoverTextColor',
+      'buttonSecondaryActiveTextColor',
+      'buttonSecondaryBorderColor',
+      'buttonSecondaryHoverBorderColor',
+      'buttonSecondaryActiveBorderColor',
+      'buttonSecondaryBackgroundColor',
+      'buttonSecondaryHoverBackgroundColor',
+      'buttonSecondaryActiveBackgroundColor',
+    ]);
+  });
+});
```

Comment:
Could argue that this test is redundant as we're importing theme attributes from `bpk-component-button`. I included it to err on the side of caution, but happy for it to go.